src/main/java/frc/robot/Robot.java

```diff
@@ -48,12 +48,14 @@
     @Override
     public void robotInit() {
         // Initialize OI here
-        Camera0 = CameraServer.getInstance().startAutomaticCapture(0);
-        Camera1 = CameraServer.getInstance().startAutomaticCapture(1);
-        Camera0.setResolution(640, 480);
-        Camera1.setResolution(640, 480);
-        videoSink = CameraServer.getInstance().getServer();
+        camera0 = CameraServer.getInstance().startAutomaticCapture(0);
+        camera1 = CameraServer.getInstance().startAutomaticCapture(1);
+        camera0.setResolution(310, 240);
```

Comment:
Out of curiosity, why 310 instead of 320?

Reply:
becuase my memory sucks and i didnt think to check it against my other number